src/data/slice.js

```diff
@@ -9,6 +9,24 @@ const slice = createSlice({
     courseId: null,
     status: null,
     canChangeProvider: null,
+    waffleFlags: {
+      useNewHomePage: true,
+      useNewCustomPages: true,
+      useNewScheduleDetailsPage: true,
+      useNewAdvancedSettingsPage: true,
+      useNewGradingPage: true,
+      useNewUpdatesPage: true,
+      useNewImportPage: false,
+      useNewExportPage: true,
+      useNewFilesUploadsPage: true,
+      useNewVideoUploadsPage: true,
+      useNewCourseOutlinePage: false,
```

Comment:
The default for this one should be `true`, as far as I can tell.

Reply:
[inform] Implemented in follow-up commit: [`5f19746` (#1372)](https://github.com/openedx/frontend-app-authoring/pull/1372/commits/5f19746fd6b2a803200abd8abdaac6adb5813fb2#diff-f708120055e193e6e1c566fefb57a208b514e2fd0f8944fc91ee1f0c97b0cd34R37)